.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,75 @@
+name: create-tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      TAG:
+        description: 'TAG'
+        required: true
+        default: ""
+        type: string
+      TAG_BODY:
+        description: 'TAG DESCRIPTION'
+        required: false
+        default: ""
+        type: string
+      IS_DRAFT:
+        description: 'CREATE AS DRAFT'
+        required: false
+        default: false
+        type: boolean
+      IS_PRERELEASE:
+        description: 'PRERELEASE TAG'
+        required: true
+        default: true
+        type: boolean
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: Install dependencies
+        run: cd synopsys-task && npm ci
+
+      - name: Rebuild the dist/ directory
+        run: cd synopsys-task && npm run build && npm run package
+
+      - name: Compare the expected and actual dist/ directories
+        run: |
+          cd synopsys-task
+          if [ "$(git diff --ignore-space-at-eol dist/ | wc -l)" -gt "0" ]; then
+            echo "Detected uncommitted changes after build.  See status below:"
+            git diff
+            exit 1
+          fi
+        id: diff
+
+      - name: replace \n
```

Comment:
is it required to add \n ?

Reply:
when we take  input in description for create tag . it taking input in one line only. it not creating the new line. so , we need to create the new line if description contains \n.